src/common/hooks/useFocused.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+const useFocused = () => {
+  const [focused, setFocused] = useState(document.activeElement);
```

Comment:
fwiw I ran into an issue the other day trying to use `document` into a `useMemo` hook and it failed because `document` did not exist yet. AFAICT this has potential to break as well unless you always use this in a `useEffect` or put the `useEffect` in this hook. Or iirc you can do a check with `if (typeof window !== 'undefined')  //then document will exist`